examples/SMAC4HPO_hyperband_mlp.py

```diff
@@ -104,9 +107,9 @@ def mlp_from_cfg(cfg, seed, instance, budget, **kwargs):
 cs.add_conditions([use_lr, use_batch_size, use_lr_init])
 
 # SMAC scenario object
-scenario = Scenario({"run_obj": "quality",     # we optimize quality (alternative runtime)
-                     "wallclock-limit": 100,   # max duration to run the optimization (in seconds)
-                     "cs": cs,                 # configuration space
+scenario = Scenario({"run_obj": "quality",      # we optimize quality (alternative runtime)
+                     "wallclock-limit": 100,    # max duration to run the optimization (in seconds)
+                     "cs": cs,                  # configuration space
                      "deterministic": "true",
                      "limit_resources": False,  # Disables pynisher to pass cutoff directly to the target algorithm.
                                                 # Timeouts have to be taken care within the TA
```

Comment:
Doesn't this contradict the logger statement above saying `"Running MLP example for SMAC. If you experience difficulties, try to decrease the memory-limit."`?